lib/modules/commentQuickCollapse.js

```diff
@@ -0,0 +1,37 @@
+/* @flow */
+
+import { $ } from '../vendor';
+import { Module } from '../core/module';
+
+export const module: Module<*> = new Module('commentQuickCollapse');
+
+module.moduleName = 'commentQuickCollapseName';
+module.category = 'commentsCategory';
+module.description = 'commentQuickCollapseDesc';
+
+module.options = {
+	hideCommentsOnHeaderDoubleClick: {
+		type: 'boolean',
+		value: true,
+		description: 'hideCommentsOnHeaderDoubleClickDesc',
+	},
+};
+
+module.go = () => {
+	if (module.options.hideCommentsOnHeaderDoubleClick.value) {
+		hideCommentsOnHeaderDoubleClick();
+	}
+};
+
+function hideCommentsOnHeaderDoubleClick() {
+	$('p.tagline').on('dblclick', (e: Event) => {
+		// the first child of p.tagline is a.expand
+		e.target.children[0].click();
```

Comment:
Could you specifically target `e.target.querySelector('.expand')` instead of `e.target.children[0]`? Just in case the layout shifts around, plus it won't add a significant performance hit.